### ISSUE TITLE
Internationalization missing in language configuration

### ISSUE THREAD

**Issue by ZoeThivet** [nvaccess/nvda#8207]
### Steps to reproduce:
Pre-requisite: use NVDA in French (may be reproduced with all languages)
> 1. Open NVDA configuration
> 2. Go to "Préférences" >"Voix" (Voice)
> 3. In the "Voix" selectbox, names of languages and countries are in English.

### Expected behavior:
> Languages and countries should be in French

### Actual behavior:
> Languages and countries are in English

### System configuration:
NVDA version:
> 2018.1.1

NVDA Installed or portable: installed

Windows version:
> Windows 8

Does the issue still occur after restarting your PC? Yes

Have you tried any other versions of NVDA?
> No

**Comment by ZoeThivet** [comment]
I would be happy to help NVDA with i10n issues (English to French), you can tell me your process

**Comment by josephsl** [comment]
Hi, what synthesizer are you using? For this combo box, NVDA will present voices according to how the synthesizer named them. Thanks.

**Comment by ZoeThivet** [comment]
I'm using eSpeak NG.

**Comment by leonardder** [comment]
I'm afraid there's nothing we can do about this. NVDA gets the list of voices from Espeak directly, without a possibility to translate them.

**Comment by leonardder** [comment]
I was partly wrong. Espeak both presents the language code and the full language name (in English). However, it looks like there are also some specific british accents, like en-gb-x-rp and en-gb-scotland, for which we can't get a translation using languageHandler.getLanguageDescription.

**Comment by zstanecic** [comment]
these are the various british dialects, but not just british accents.

wm is for example west midlands.



W dniu 09.05.2018 o 17:05, Leonard de Ruijter pisze:

**Comment by seanbudd** [comment]
Is this still an issue in 2025.2?

**Comment by tareh7z** [comment]
@seanbudd Yes, the issue is still present in 2025.2 and can still be observed under the same conditions as before.

**Comment by zstanecic** [comment]
And it will never be fixed. This is the espeak's problem. What do you expect from a tts, which focuses on quantity than quality when it comes to language support? Recently, we have seen new languages in Espeak growing. I am wondering who reviews and maintains all this? 
Also, language names in Espeak are hardcoded in the text files...

**Comment by seanbudd** [comment]
If this is an upstream issue with eSpeak has this been reported there? It would be good to link the issue from this ticket
